d3/hyper.js

```diff
@@ -185,51 +205,11 @@ function generateDisplayNames(treeData) {
                 }
                 break;
         }
-    }, function(n) {
-        return n.children;
-    });
-}
-
-function collapseNodes(treeData, graphCollapse) {
-    var streamline = graphCollapse === "s" ? common.streamline : common.collapseChildren;
-    var collapseChildren = common.collapseChildren;
-    if (graphCollapse !== 'n') {
```

Comment:
Are we de-supporting the graphCollapse options for Hyper plans?

Reply:
While I personally don't use the `graphCollapse`, I adjusted the commit so that we keep supporting `graphCollapse`

Reply:
Great!